shell/shared/demo.js

```diff
@@ -264,8 +264,9 @@ Router.map(function () {
       var appName = 'missing package';
 
       if (thisPackage) {
-        var actionTitle = thisPackage.manifest.actions[0].title.defaultText;
-        appName = appNameFromActionName(actionTitle);
+        var manifest = thisPackage.manifest;
+        var action = manifest.actions[0];
+        appName = (manifest.appTitle && manifest.appTitle.defaultText) || appNameFromActionName(action.title.defaultText);
```

Comment:
I'll fix this, but note that the style guide calls for lines to be wrapped at 100 chars.

https://github.com/sandstorm-io/capnproto/blob/master/style-guide.md#spacing-and-bracing